setup.py

```diff
@@ -1,12 +1,23 @@
-from setuptools import setup
+import io
+import os
+from setuptools import setup, find_packages
+from tune_sklearn import __version__
+
+ROOT_DIR = os.path.dirname(__file__)
+
+VERSION = os.environ.get("TSK_RELEASE_VERSION", __version__)
```

Comment:
Why are we checking for this environment variable?

Reply:
makes it easier to do release candidates / dryruns :)